pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[project]
+name = "vehicle-simulation-environment"
+dynamic = ["version"]
```

Comment:
You're missing `__version__` in a Python file for this to work. Just omit the version field for now.